templates/smartstart/index.html

```diff
@@ -0,0 +1,508 @@
+{% extends "smartstart/base_smartstart.html" %}
+
+
+{% block title %}Smart things as a service{% endblock %}
```

Comment:
the title should be "SMART START - Fast-track to market with Ubuntu Core" as per the title of the copy doc.